README.md

```diff
@@ -91,7 +91,7 @@ Then access the site at <http://localhost:8080/healthcheck>
 Once you have merged your changes into master branch, deploying them is made up of
 two steps:
 
-- Pushing a built image to the docker registry from Jenkins.
+- Pushing a built image to the docker registry.
```

Comment:
Don't you need to click the 'Deploy to production' button too?

Reply:
you do, but given this is covered in the over-arching deployment docs, I feel it's an unnecessary detail for in here

Reply:
I'll resolve this as we can always refine the docs.